components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.h

```diff
@@ -13,8 +13,27 @@
 // limitations under the License.
 
 #import <UIKit/UIKit.h>
+#import "MDCBottomDrawerState.h"
 
+@class MDCBottomDrawerContainerViewController;
 @protocol MDCBottomDrawerHeader;
+@protocol MDCBottomDrawerContainerViewControllerDelegate;
+
+/**
+ Delegate for MDCBottomDrawerContainerViewController.
+ */
+@protocol MDCBottomDrawerContainerViewControllerDelegate <NSObject>
+/**
+ This method is called when the bottom drawer will change its presented state to one of the
+ MDCBottomDrawerState states.
+
+ @param containerViewController the container view controller of the bottom drawer.
+ @param drawerState the drawer's state.
+ */
+- (void)bottomDrawerContainerViewControllerWillChangeState:
```

Comment:
Please add test/tests for the changes to this file, including the state property and delegate callbacks.